### PR TITLE
Json parser

### PR DESCRIPTION
**Pull request by DanMargera**
Implementing issue #30 .
Please do not Review/Merge until requested, as this pull request still has work in progress.

### CODE REVIEW

File: app/src/main/java/sistemas/puc/com/finantialapp/FetchRatesTask.java [hunk 1]
```diff
@@ -90,4 +101,27 @@ protected void onProgressUpdate(Void... voids) {}
...
+
+        Iterator<String> rates = ratesJson.keys();
+
+        while (rates.hasNext()) {
+            String code = rates.next();
+            String name = "NameFromHash"; // TODO: create a hashtable to get the name from code.
+            double rate = ratesJson.getDouble(code);
+            // Temporarily using a placeholder Date before
+            // deciding if date should even be in MoedaItem.
+            result.add(new MoedaItem(code, name, rate, Util.getTimeFromDate(7,10,2017)));
```
Comment (DanMargera) [inline app/src/main/java/sistemas/puc/com/finantialapp/FetchRatesTask.java]: @tabocu Should each MoedaItem really contain the date? Or should I remove it from our data structure?

Comment (tabocu) [reply app/src/main/java/sistemas/puc/com/finantialapp/FetchRatesTask.java]: Ass discussed, we will keep that way.

**Review by tabocu — DISMISSED** [review]
Please, add map before merging it.

**Comment by DanMargera** [comment]
Closes #30

**Review by tabocu — APPROVED** [review]

**Pull request merged**